src/components/sidebar/routePathView/NewRoutePathView.tsx

```diff
@@ -41,13 +43,33 @@ class NewRoutePathView extends React.Component<INewRoutePathViewProps, INewRoute
             };
         } else {
             this.state = {
-                currentRoutePath,
+                currentRoutePath: { ... currentRoutePath },
             };
         }
 
         this.initStores();
     }
 
+    componentWillMount() {
+        this.getDateSuggestion();
+    }
+
+    public async getDateSuggestion() {
+        const currentRoutePath = this.state.currentRoutePath;
+        if (currentRoutePath) {
+            const results = await RouteService.fetchRoute(currentRoutePath.routeId);
```

Comment:
const route = await ...

yes it was my code, but didn't think when I wrote it...